### PR TITLE
Fix ServiceEntry config definition for Mesh Expansion

### PR DESCRIPTION
**Pull request by pitlv2109**

### CODE REVIEW

File: content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md [hunk 1]
```diff
@@ -266,12 +266,15 @@ The `server: envoy` header indicates that the sidecar intercepted the traffic.
     {{< text bash >}}
     $ istioctl  register -n ${SERVICE_NAMESPACE} vmhttp ${GCE_IP} 8080
     {{< /text >}}
+    
+    Make sure you have already added `istioctl` client to your PATH environment variable, as described
+    in step 3 in the [Download page](/docs/setup/kubernetes/download/#download-and-prepare-for-the-installation).
```
Comment (incfly) [inline content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md]: Should we move this up before `istioctl` command?

**Comment by pitlv2109** [comment]
The circleci test seems to be flaky. Many recent PRs failed as well.

File: content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md [hunk 2]
```diff
@@ -135,13 +135,12 @@ Next, run the following commands on each machine that you want to add to the mes
...
     {{< text bash >}}
-    $ gcloud compute ssh --project=${MY_PROJECT} --zone={MY_ZONE} "${GCE_NAME}"
+    $ gcloud compute ssh --project=${MY_PROJECT} --zone=${MY_ZONE} "${GCE_NAME}"
     $ curl -L https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/deb/istio-sidecar.deb > istio-sidecar.deb
     $ sudo dpkg -i istio-sidecar.deb
     {{< /text >}}
 
-1.  Add the IP address of the Istio gateway to `/etc/hosts`. Revisit [the preparing the cluster section](#preparing-the-kubernetes-cluster-for-expansion) to learn how to obtain the IP address.
-to `/etc/hosts` or to
+1.  Add the IP address of the Istio gateway to `/etc/hosts`. Revisit [the preparing the cluster section](#preparing-the-kubernetes-cluster-for-expansion) to learn how to obtain the IP address.
```
[the remark this answers is not in the document]
Comment (pitlv2109) [reply content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md]: Done. Thanks!

File: content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md (hunk 1 above)
Comment (pitlv2109) [reply content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md]: Done. Also put PATH in backticks in the Download page.

**Comment by pitlv2109** [comment]
I have a linting error, but circleci does not provide any helpful info :(

**Comment by geeknoid** [comment]
From the CircleCi log:

>> 352 files are free from spelling errors
./content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md:143: MD009 Trailing spaces
./content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md:270: MD009 Trailing spaces

**Pull request merged**